src/Orchard.Web/Modules/Orchard.Workflows/Controllers/AdminController.cs

```diff
@@ -303,6 +303,19 @@ IFormManager formManager
                 });
             }
 
+            foreach (var workflowRecord in workflowDefinitionRecord.WorkflowRecords) {

```

Comment:
@jtkech ok I see. Perhaps the dialog should be asking 'You have running instances of this workflow, do you want to stop them?' and then pass their response. For a small change, such as a change of email content, you do not want your long running workflows to be deleted.